packages/renderers/renderer-lit/server.js

```diff
@@ -28,10 +36,27 @@ function* render(tagName, attrs, children) {
   const instance = new LitElementRenderer(tagName);
 
   // LitElementRenderer creates a new element instance, so copy over.
+  const Ctr = getCustomElementConstructor(tagName);
   for (let [name, value] of Object.entries(attrs)) {
-    instance.setAttribute(name, value);
+    // check if this is a property reserved by JSX parser
+    if (reservedReactProperties.has(name)) {
+      console.warn(
+        `${tagName} contains property ${name} which is a ` +
+          `reserved property. It will be used by Astro and not set on ` +
```

Comment:
Is there a link this can point to for more information? It took me a bit to grok this and only clicked once I looked at the reserved property list.

Reply:
No idea